api/src/main/java/org/commonjava/indy/model/galley/CacheOnlyLocation.java

```diff
@@ -50,13 +57,16 @@ public CacheOnlyLocation( final HostedRepository repo )
 
     public CacheOnlyLocation( final StoreKey key )
     {
-        this.repo = null;
+        this.isHosted = false;
+        this.isAllowReleases = true;
```

Comment:
Not understand here, if it is not hosted, why release is still allowed? (For group case?) 
Or maybe my mistake on understanding, the release here means we can release artifact to this location(repo), or it represents this location is a "release" but not "snapshot" artifact?

Reply:
Yep, it's for the group case. There's another kind of Location, RepositoryLocation, for remote repos.

Reply:
Ok, got that.